src/Types.jl

```diff
@@ -328,7 +328,9 @@ Base.@kwdef mutable struct Context
     use_only_tarballs_for_downloads::Bool = false
     num_concurrent_downloads::Int = 8
     graph_verbose::Bool = false
-    currently_running_target::Bool = false
```

Comment:
I didn't want this to be confused with the `julia_version` since they sound almost similar; and since this flag doesn't seem to be used anymore (since we have dropped support for the Pkg 2 feature that used this) it seemed fine to remove it.